cloudflare/resource_cloudflare_load_balancer_monitor_test.go

```diff
@@ -265,6 +265,15 @@ resource "cloudflare_load_balancer_monitor" "test" {
 }`
 }
 
+func testAccCheckCloudflareLoadBalancerMonitorConfigEmptyExpectedBody() string {
```

Comment:
This test doesn't actually assert anything as is. We'll need to create a test case similar to https://github.com/terraform-providers/terraform-provider-cloudflare/pull/448/files#diff-ff2414310a6fa415074bb1da98684b38R91-R104 for asserting that this resource is created and looks like we'd expect it to.

Reply:
Thanks! Will try and push a true assertion there soon.